src/contexts/ProgressContext.tsx

```diff
@@ -117,12 +119,15 @@ export const useProgramContentProgress = (programId: string, memberId: string) =
         ? undefined
         : flatten(
             data.program_content_body.map(contentBody =>
-              contentBody.program_contents.map(content => ({
-                programContentId: content.id,
-                programContentSectionId: content.content_section_id,
-                progress: content.program_content_progress[0]?.progress || 0,
-                lastProgress: content.program_content_progress[0]?.last_progress || 0,
-              })),
+              contentBody.program_contents.map(content => {
+                return {
+                  programContentType: contentBody.type,
```

Comment:
this type is result from program_content_body, prefer the variable name is programCotnentBodyType